.env.example

```
@@ -58,3 +58,6 @@ WORKER_CONCURRENCY=50
 AI_ENABLED=true
 AI_FLOWS_CONFIG='{"custom":{"success_flow":{"adapter":"flowise","predictionUrl":"http://localhost:57657/test/flowise/api/v1/prediction/success"}, "failed_flow":{"adapter":"flowise","predictionUrl":"http://localhost:57657/test/flowise/api/v1/prediction/fail/500"}, "success_n8n_flow":{"adapter":"n8n","predictionUrl":"http://localhost:57657/test/n8n/webhook/success"}, "failed_n8n_flow":{"adapter":"n8n","predictionUrl":"http://localhost:57657/test/n8n/webhook/fail/500"}}}'
 AI_ADAPTERS_CONFIG='{"flowise":{"secret":"test-no-real-secret"}, "n8n":{"apiKey":"test-n8n-api-key"}}'
+
+# Subscriptions
+ENABLE_SUBSCRIPTIONS=true
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)